src/helpers/script_helpers.sh

```diff
@@ -109,9 +109,15 @@ function run {
     undo_command
   else
     remove_step_files
+
+    # Running inside a subfolder can cause issues if that folder doesn't exist
+    # on the main branch. Hence we cd into the main branch here.
```

Comment:
Comment doesn't make sense to me. I don't believe it has anything to do with the main branch and just depends if the folder is known to git yet or not. And you aren't "cd"ing into the main branch but into the root git folder